code/src/components/categorySection/CategorySection.tsx

```diff
@@ -0,0 +1,49 @@
+import Box from "@mui/material/Box";
+import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
+import AppTypography from "@/components/app-typography/AppTypography";
+import categoryData from "@/components/categorySection/CategorySection.constants";
+import "@/components/categorySection/CategorySection.scss";
+
+type CategoryElementProps = {
+  label: string;
+  image: string;
+};
+
+const CategoryElement = ({ label, image }: CategoryElementProps) => {
```

Comment:
move it to separate components and name it CategoryItem

Reply:
Done